### PR TITLE
Settings for coding convention

### PR DESCRIPTION
**Pull request by btry**
@DIOHz0r please can you add settings for your IDE ? (phpStorm I guess) push a commit on this PR

If we have similar settings for VS Code or other, this is welcome.

### CODE REVIEW

**Comment by btry** [comment]
@DIOHz0r Any instruction to import the settings for PHPStorm ?

**Comment by DIOHz0r** [comment]
Now the guide was updated

File: tools/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+Title:       Content of the folder  
+Subtitle:    Describe the purpose of each file  
+Project:     Flyve MDM plugin for GLPI
+Author:      Thierry Bugier <tbugier@teclib.com>  
+---
```
Comment (ajsb85) [inline tools/README.md]: plus contributor: Domingo Oropeza

Comment (DIOHz0r) [reply tools/README.md]: updated.

**Pull request merged**